Mailer/app/api/Validation/schemas/logs.schema.json

```diff
@@ -0,0 +1,178 @@
+                "type": "string",
+                "format": "date"
+              },
+              "to": {
+                "title": "To filter",
+                "type": "string",
+                "format": "date"
+              }
+            },
+            "additionalProperties": false
+          },
+          "clicked_at":{
+            "type": "object",
+            "title": "Clicked at filter",
+            "properties": {
+              "from": {
+                "title": "From filter",
+                "type": "string",
+                "format": "date"
+              },
+              "to": {
+                "title": "To filter",
+                "type": "string",
+                "format": "date"
+              }
+            },
+            "additionalProperties": false
+          },
+          "opened_at":{
+            "type": "object",
+            "title": "Opened at filter",
+            "properties": {
+              "from": {
+                "title": "From filter",
+                "type": "string",
+                "format": "date"
+              },
+              "to": {
+                "title": "To filter",
+                "type": "string",
+                "format": "date"
+              }
+            },
+            "additionalProperties": false
+          },
+          "dropped_at":{
+            "type": "object",
+            "title": "Dropped at filter",
+            "properties": {
+              "from": {
+                "title": "From filter",
+                "type": "string",
+                "format": "date"
+              },
+              "to": {
+                "title": "To filter",
+                "type": "string",
+                "format": "date"
+              }
+            },
+            "additionalProperties": false
+          },
+          "spam_complained_at":{
+            "type": "object",
+            "title": "Spam complained at filter",
+            "properties": {
+              "from": {
+                "title": "From filter",
+                "type": "string",
+                "format": "date"
+              },
+              "to": {
+                "title": "To filter",
+                "type": "string",
+                "format": "date"
+              }
+            },
+            "additionalProperties": false
+          },
+          "hard_bounced_at":{
+            "type": "object",
+            "title": "Hard bounced at filter",
+            "properties": {
+              "from": {
+                "title": "From filter",
+                "type": "string",
+                "format": "date"
+              },
+              "to": {
+                "title": "To filter",
+                "type": "string",
+                "format": "date"
+              }
+            },
+            "additionalProperties": false
+          }
+        },
+        "additionalProperties": false
+      }]
+    },
+    "limit": {
+      "type": "integer",
+      "maximum": 300
```

Comment:
This limit seems too arbitrary. Let's remove it. Client could request for less/more and I can't see any reasoning why stop at 300.

Reply:
Previously you said that there can be too many logs in DB and that filter should be required. In CRM however, there is a need for unfiltered logs so I removed the condition of the required filter and instead of it I set a limit to number of results.